src/frontends/pytorch/src/transforms/tuple_unpack_replacer.cpp

```diff
@@ -29,6 +29,12 @@ PrimTupleUnpackReplacer::PrimTupleUnpackReplacer() {
         auto input_node = tuple_unpack->get_input_node_shared_ptr(0);
         auto tuple_construct = cast_fw_node(input_node, "prim::TupleConstruct");
         if (!tuple_construct) {
+            if(!ov::as_type_ptr<ov::op::util::FrameworkNode>(input_node)) {
```

Comment:
This condition is too rough: it breaks cases where a tuple comes not from FrameworkNode but still is handled somewhere else in a more accurate way. So we need to apply the code below with bypassing outputs only when we really know the op that produces it -- in our case, it is an op produced by `inlined_extension`.